src/hooks/useMatomo.ts

```diff
@@ -0,0 +1,30 @@
+import Matomo from "matomo-tracker"
+import { useMemo, useCallback } from "react"
+import { TrackingService } from "./usePlausible"
+
+let _matomo: any
+
+const DEBUG = process.env.NEXT_PUBLIC_DEPLOY_ENV !== "production"
```

Comment:
We are using the new `NEXT_PUBLIC_DEPLOY_ENV` variable to determine if the service should be live or not.